grantnav/frontend/tests_functional.py

```diff
@@ -418,3 +418,8 @@ def test_currency_facet(provenance_dataload, server_url, browser):
     browser.find_element_by_class_name("large-search-icon").click()
     browser.find_element_by_link_text("USD (4)").click()
     assert 'USD 0 - USD 500' in browser.find_element_by_tag_name('body').text
+
+@pytest.mark.parametrize(('path'), ['/grant/360G-wolfson-19916'])
+def test_zero_grant_info_link(provenance_dataload, server_url, browser, path):
+    browser.get(server_url + path)
+    assert 'icon-help-50.png' in find_element_by_tag_name('body').text
```

Comment:
Could you also check that the link redirects to the correct place?
Could you also add another test that tests that the icon is not there when the amount is not 0?